R/info.R

```diff
@@ -77,10 +81,22 @@ check_feature = function(feature_name, context = NULL, call = sys.call(1L)) {
 #'
 #' The threadpool size can be overridden by setting the
 #' `POLARS_MAX_THREADS` environment variable before process start.
-#' (The thread pool is not behind a lock, so it cannot be modified once set).
+#' It cannot be modified once `polars` is loaded.
 #' It is strongly recommended not to override this value as it will be
 #' set automatically by the engine.
 #'
+#' For compatibility with CRAN, the threadpool size is set to 2 by default.
+#' To disable this behavior and let the engine determine the threadpool size,
+#' one of the following ways can be used:
+#'
+#' - Enable the `disable_auto_limit_max_threads` feature of the library.
```

Comment:
for consistency I think this feature should also be renamed `limit_max_threads` on the Rust side

Reply:
You mean "disable_limit_max_threads"?
Note that this feature must be turned off by default (must not be included in the default feature)

Reply:
Oh yes I have inverted it in my mind. I think the choices should be:

- enable the feature `disable_limit_max_threads` (or even `no_limit_max_threads`?)
- `options(polars.limit_max_threads = FALSE)`

Reply:
Renamed to `disable_limit_max_threads`